SWI-cpp2.h

```diff
@@ -583,6 +582,34 @@ class PlTerm_recorded : public PlTerm
   explicit PlTerm_recorded(record_t r) { PlCheck(PL_recorded(r, C_)); }
 };
 
+class PlModule : public WrappedC<module_t>
+{
+public:
```

Comment:
Here we may  have a null constructor.   Practically all API functions accept a NULL module as a shorthand for the current calling context, defaulting to `user` if such a context does not exist (when we make calls to Prolog without being called by Prolog).

Reply:
I've added a constructor that takes module_t, so PlModule(PlModule::null) is now possible.